### PR TITLE
Hi ma'am, i have added more screenshots of app, please merge it

### PR DESCRIPTION
**Pull request by amitYadavDev**

### CODE REVIEW

**Comment by saumya1singh** [comment]
@amit6663  great work

**Comment by saumya1singh** [comment]
just few more changes and you will earn your first reward .

File: README.md [hunk 1]
```diff
@@ -9,14 +9,97 @@ Step 2 : Open project in Android Studio
...
 
-# Here are some screenshots of the application
-
-![Picture1 (2)](https://user-images.githubusercontent.com/46818757/69555788-bc498780-0fc9-11ea-90dd-607153545e01.png)
-![Picture2 (2)](https://user-images.githubusercontent.com/46818757/69555841-d08d8480-0fc9-11ea-8980-f583356f8346.png)
-![Picture3 (2)](https://user-images.githubusercontent.com/46818757/69555856-d5eacf00-0fc9-11ea-9f27-7244da771395.png)
-![Picture4 (2)](https://user-images.githubusercontent.com/46818757/69555891-df743700-0fc9-11ea-8c77-3e047e550b07.png)
-![Picture5 (2)](https://user-images.githubusercontent.com/46818757/69555903-e3a05480-0fc9-11ea-87b4-241b0e210c1e.png)
-![Picture6 (2)](https://user-images.githubusercontent.com/46818757/69555910-e8650880-0fc9-11ea-9945-070633fd7098.png)
+# Here are all screenshots of the application
```
Comment (saumya1singh) [inline README.md:12]: change all to some @amit6663

File: README.md [hunk 2]
```diff
@@ -9,14 +9,97 @@ Step 2 : Open project in Android Studio
...
+
+
+![Screenshot_20191216-211815](https://user-images.githubusercontent.com/45551012/70924765-19c47780-2050-11ea-8f4f-26777ae34a2e.png)
+
+
+![Screenshot_20191216-211848](https://user-images.githubusercontent.com/45551012/70924776-234ddf80-2050-11ea-8fd8-9e34390c7bad.png)
+
+
+![Screenshot_20191216-211928](https://user-images.githubusercontent.com/45551012/70924805-2ea10b00-2050-11ea-8562-5087a2f5b136.png)
+
```
Comment (saumya1singh) [inline README.md:17]: @amit6663  change the image sizes . it's looking quite big . make it look small and concised .

**Review by saumya1singh — CHANGES_REQUESTED** [review]

**Comment by saumya1singh** [comment]
@amit6663  well done , your work is very appreciating 🎉

**Comment by saumya1singh** [comment]
@amit6663  I will approve your changes just after a quick review .

**Comment by saumya1singh** [comment]
@amit6663  I want you to remove some of the screens from screenshots that are not well designed.
I am writing the picture number which you have to remove : 7,11,12,13,14,16,17,18,19,23,24,28 ,26,27 .

When you guys will work on improving the screens then we will add more SS . 

And last thing is  squash your commits into one commit .

Then i will approve your changes and you will earn your point for KOSS contest .

**Comment by amitYadavDev** [comment]
I have done ma'am please look again, @SaumyaSingh1

**Comment by saumya1singh** [comment]
You will be rewarded once I approve @amit6663

**Review by saumya1singh — APPROVED** [review]

**Comment by saumya1singh** [comment]
@amit6663 approved 🎉

**Pull request closed without merging**